src/clients/SpokePoolClient.ts

```diff
@@ -1098,4 +1125,80 @@ export class SpokePoolClient extends BaseAbstractClient {
 
     return deposit;
   }
+
+  async findDepositV3(depositId: number, destinationChainId: number, depositor: string): Promise<V3DepositWithBlock> {
+    // Binary search for block. This way we can get the blocks before and after the deposit with
```

Comment:
This comment looks like it should have been removed when the original functionality was copied - there's no binary search here. ooc, do you intend for this to revert to a binary search in future?

Reply:
Replaced the language with "binary search for event query search bounds" which I think is more accurate